basxconnect/core/views/person/person_browse_views.py

```diff
@@ -31,6 +39,112 @@ def bulkrestore(request, qs):
         person.save()
 
 
+def bulk_tag_operation(request):
```

Comment:
For this view: I think you could use ```bread.utils.urls.aslayout``` (which in the meantime seems to be the wrong package for that decorator) as a decorator for the view, to avoid the whole import-string magic.

Reply:
Hmm but I also want to be able to return `HttpResponseRedirect` or `HttpResponseBadRequest`...
Can I do that with `@aslayout`?

Reply:
Good point. I updated bread quickly to consider non-layout responses, I think that makes sense anyway (is pushed but not released)